bundlewrap/group.py

```diff
@@ -61,10 +61,11 @@
     'bundles': COLLECTION_OF_STRINGS,
     'cmd_wrapper_inner': str,
     'cmd_wrapper_outer': str,
-    'lock_dir': str,
     'dummy': bool,
     'file_path': str,
+    'ipmi': dict,
```

Comment:
Sorry, I only saw this now: Usage of the new `ipmi` attribute isn’t documented yet, right?